README.md

```diff
@@ -9,69 +9,51 @@
 
 ### [Website](https://seed-rs.org) | [Forum](https://seed.discourse.group) | [Chat](https://discord.gg/JHHcHp5)
 ---
-Seed is a Rust framework for creating fast and reliable web apps with an elm-like architecture.
+Seed is a front end Rust framework for creating fast and reliable web apps with an elm-like architecture.
 
-- Minimal boilerplate, overhead, and configuration.
+- All the benefits of rust and macro based syntax.
+- `wasm-bindgen` zero-cost performance, with no JavaScript required.
+- Minimal overhead, configuration, and boilerplate.
 - Clear documentation made to be accessible regardless of your familiarity with Rust.
-- Safe code.
+- Written without any [unsafe](https://doc.rust-lang.org/book/ch19-01-unsafe-rust.html) code and works with `stable` Rust, no `nightly` required!
+
+[More](#why-use-seed) on why you should use Seed.
 
 ---
 
 # Quickstart
 
 If you are proficient in a front end framework, creating a standalone web app is painless.
 
-To get started, you can clone our quickstart. We explain in detail [here](https://seed-rs.org/guide/quickstart).
-
-<details>
-<summary>seed-quickstart</summary>
-
-[Quickstart repository](https://github.com/seed-rs/seed-quickstart) | [Download Rust](https://www.rust-lang.org/tools/install)
-~~~
-rustup update
-rustup target add wasm32-unknown-unknown
-cargo install --force cargo-make
-~~~
-~~~
-git clone https://github.com/david-oconnor/seed-quickstart.git
-cd seed-quickstart
-cargo make build
-cargo make serve
-~~~
-To build automatically use `cargo make watch`.
-</details>
+To get started, you can clone our [quickstart](https://github.com/seed-rs/seed-quickstart), where we explain in detail.
```

Comment:
Do you think we should link to the quickstart repo and just mirror it on the website, or link to the website where we'd write the `git clone` code and whatnot so the user doesn't have to navigate through multiple links?

Reply:
I would point user directly to quickstart repo. I think the user will jump to quickstart repo if we want or not. Also there is a smaller risk that he would read out-dated instructions and we don't have analytics on the website (yet) - github's analytics are good enough for now. And there is a chance he star the repo.
If you want you can also add something like "super mega new Rust-only quickstart in development" and add link to Seeder.